test/testConfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "files": ["test/example.js"],
   "customLinks": {
+    "ExampleClass": "#class-exampleclass",
```

Comment:
Could we remove this for now, and add support for automatic generation of class links in a separate PR? (where appropriate links will appear)